libraries/mbed/targets/cmsis/TARGET_NXP/TARGET_LPC81X/system_LPC8xx.h

```diff
@@ -30,6 +30,8 @@ extern "C" {
 
 #include <stdint.h>
 
+//WH
```

Comment:
Why are you leaving this comments in the code? We can "blame" to get that detail.